lib/models/purchase_params.dart

```diff
@@ -39,7 +42,11 @@ class PurchaseParams {
   /// for more info.
   ///
   /// [promotionalOffer] iOS only. Promotional offer that will be applied to the product.
-  /// Retrieve this offer using [Purchases.getPromotionalOffer].
+  /// Retrieve this offer using [Purchases.getEligibleWinBackOffersForPackage].
```

Comment:
I think this is incorrect?
```suggestion
  /// Retrieve this offer using [Purchases.getPromotionalOffer].
```

Reply:
In addition, perhaps I would add
> Ignored if a win-back offer is provided

Reply:
If so, we should probably add the same phrase to the other `PurchaseParams` constructors docs

Reply:
Totally right, fixed in https://github.com/RevenueCat/purchases-flutter/pull/1474/commits/f92b28fcb863caa01491e9f386efe07b810416ad